code/modules/mob/living/simple_animal/friendly/schlorrgo.dm

```diff
@@ -247,7 +250,7 @@
 			a_intent = I_HURT
 			emote_sounds = list('sound/effects/creatures/schlorrgo_scream.ogg')
 			holder_type = null
-			max_eggs = 9
+			max_eggs = 40
```

Comment:
PR Description says 30.
This says 40.
What's the intended value?

Reply:
My bad, forgot to mention this. So this is max fat schlorrgo, and part of it has it so a 75% ish chance that any eggs laid are destroyed immediately. I raised the max to have this occur